include/pros/imu.hpp

```diff
@@ -206,8 +206,173 @@ class Imu {
 	 *
 	 * \param  port
 	 * 				 The V5 Inertial Sensor port number from 1-21
-	 * \return The raw accelerometer values. If the operation failed, all the
-	 * structure's members are filled with PROS_ERR_F and errno is set.
+	 * \return 1 if the operation was successful or PROS_ERR if the operation
+	 * failed, setting errno.
+	 */
+	virtual std::int32_t reset_rotation() const;
```

Comment:
I know it's existing practice, but I was never a fan of these "constant" functions that mutated things.

Reply:
Probably a thing for later though.